src/main/java/graphql/execution/ExecutionStrategyParameters.java

```diff
@@ -50,35 +50,37 @@ public NonNullableFieldValidator nonNullFieldValidator() {
     }
 
     public ExecutionStrategyParameters transform(Consumer<Builder> builderConsumer) {
-        Builder builder = newParameters(this);
+        Builder builder = new Builder(this);
         builderConsumer.accept(builder);
         return builder.build();
     }
 
-    public static Builder newParameters() {
-        return new Builder();
-    }
-
-    public static Builder newParameters(ExecutionStrategyParameters oldParameters) {
```

Comment:
My interpretation of the pattern was that `ExecutionStrategyParameters` had access to the constructors of `Builder`, while outside classes do not. Given that, `ExecutionStrategyParameters#transform(Consumer)` could call the constructor directly instead of via an additional layer of indirection (the static accessor of the constructor). There were no external callers of `ExecutionStrategyParameters#newParameters(ExecutionStrategyParameters)`, so it was safe to delete.

What's the reason for using this method?